.circleci/config.yml

```diff
@@ -138,10 +138,6 @@ step_rubocop: &step_rubocop
     # The workaround is to use `cpu.shares / 1024`:
     # https://discuss.circleci.com/t/environment-variable-set-to-the-number-of-available-cpus/32670/4
     command: PARALLEL_PROCESSOR_COUNT=$((`cat /sys/fs/cgroup/cpu/cpu.shares` / 1024)) bundle exec rake rubocop
-step_sorbet_type_checker: &step_sorbet_type_checker
-  run:
-    name: Run sorbet type checker
-    command: bundle exec rake typecheck
```

Comment:
(Adding this for future reference) See https://github.com/DataDog/dd-trace-rb/issues/2641 for details on why we're removing Sorbet.